python/orca/src/bigdl/orca/learn/mpi/mpi_estimator.py

```diff
@@ -139,8 +142,11 @@ def fit(self, data, epochs=1, batch_size=32, validation_data=None, validate_batc
                               validate_batch_size, train_func, validate_func, train_batches,
                               validate_batches, validate_steps), f)
         self.mpi_runner.scp_file("mpi_train_data.pkl", self.dir)
-        self.mpi_runner.run("{}/mpi_train.py".format(self.dir), pkl_path=self.dir)
-        self.mpi_runner.shutdown_plasma()
+        self.mpi_runner.run("{}/mpi_train.py".format(self.dir),
+                            mpi_options=mpi_options,
+                            pkl_path=self.dir)
+        if isinstance(data, DataFrame) or isinstance(data, RDD):
```

Comment:
You also need to add this if when launch_plasma

Reply:
launch_plasma  is already under the condition if isinstance(data, DataFrame)

Reply:
ok. Then I think we can remove RDD, since RDD is not supported in fit methods of all estimators.

Reply:
Actually after training, the input 'data' which was DataFrame has been changed to RDD. When we want to shutdown plasma,  the data is RDD. Will change to if isinstance(data, RDD): self.mpi_runner.shutdown_plasma()

Reply:
I don't think it is reasonable to change the original data format.

Reply:
updated